### PR TITLE
[usage] Save inserted and updated records in the DB

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

Extends control loop to save records in the DB.

## Related Issue(s)

Fixes #

## How to test

Unit tests

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-preview

### CODE REVIEW

File: components/usage/pkg/apiv1/usage.go [hunk 1]
```diff
@@ -344,6 +344,24 @@ func (s *UsageService) ReconcileUsageWithLedger(ctx context.Context, req *v1.Rec
 	inserts, updates := reconcileUsageWithLedger(instances, usageDrafts, s.pricer, now)
 	logger.WithField("inserts", inserts).WithField("updates", updates).Infof("Identified %d inserts and %d updates against usage records.", len(inserts), len(updates))
 
+	if len(inserts) > 0 {
+		err = db.InsertUsage(ctx, s.conn, inserts...)
+		if err != nil {
+			logger.WithError(err).Errorf("Failed to insert %d usage records into the datbase.", len(inserts))
```
Comment (svenefftinge) [inline components/usage/pkg/apiv1/usage.go:350]: nit: typo 'datbase'

**Review by svenefftinge — APPROVED** [review]

File: components/usage/pkg/apiv1/usage.go (hunk 1 above)
Comment (easyCZ) [reply components/usage/pkg/apiv1/usage.go]: thanks, fixed.

**Pull request merged**